project/routes/web.php

```diff
@@ -60,6 +60,7 @@
     Route::get('edituser/{id}', [\App\Http\Controllers\Auth\RegisteredUserController::class, "edituser"])->name("edituser");
     Route::post('update/{id}', [\App\Http\Controllers\Auth\RegisteredUserController::class, "update"])->name("update");
     Route::post('delete/{id}', [\App\Http\Controllers\Auth\RegisteredUserController::class, "delete"])->name("delete");
+    Route::post('update_item/{id}', [\App\Http\Controllers\StorageController::class, "update"])->name('update');
```

Comment:
Czy to nie będzie się gryzło z Route w linijce 61? Może nazwa "updateItem" byłaby lepsza

Reply:
Zmienię